speech/start.py

```diff
@@ -1,45 +1,56 @@
 #!/usr/bin/env python
 
-from flask import Flask, jsonify, request, make_response
+import os
+import sys
+from flask import Flask, jsonify, send_file
 from flask_cors import CORS
 from urllib.parse import unquote
+from io import BytesIO
 import speechManager
 import config
 
+# Add the parent directory to the Python path
+current_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if current_dir not in sys.path:
+    sys.path.insert(0, current_dir)
+
 app = Flask(__name__)
 app.url_map.strict_slashes = False
 CORS(app)
 
-@app.route('/voices/', methods=['GET'])
+
+@app.route("/voices/", methods=["GET"])
 def voices():
     voices = speechManager.getVoices()
     return jsonify(voices)
 
 
-@app.route('/speak/<text>/', methods=['POST', 'GET'])
-@app.route('/speak/<text>/<providerId>/', methods=['POST', 'GET'])
-@app.route('/speak/<text>/<providerId>/<voiceId>', methods=['POST', 'GET'])
-def speak(text, providerId="", voiceId=""):
+@app.route("/speak/<text>/", methods=["POST", "GET"])
+@app.route("/speak/<text>/<providerId>/", methods=["POST", "GET"])
+@app.route("/speak/<text>/<providerId>/<voiceId>", methods=["POST", "GET"])
+def speak_text(text: str, providerId: str = "", voiceId: str = ""):
```

Comment:
didn't know Python has TypeScript-like types :D

Reply:
yeah - some hate it!